### PR TITLE
Scope loading behaviour instead of using early return

### PR DESCRIPTION
**Pull request by aryeguy**
Fails under babel_node node_modules/mocha/bin/_mocha
Transformation error; return original code
{ [SyntaxError: node_modules/mocha/bin/_mocha: 'return' outside of function (392:2)]
  pos: 9902,
  loc: { line: 392, column: 2 },
  raisedAt: 9908,
  _babel: true,
  codeFrame: '  390 |   });\n  391 | \n> 392 |   return;\n      |   ^\n  393 | }\n  394 | \n  395 | // load' }

based on advice from @oakfang

### CODE REVIEW

File: bin/_mocha [hunk 1]
```diff
@@ -392,15 +392,15 @@ if (program.watch) {
...
 
 // load
 
-mocha.files = files;
-runner = mocha.run(program.exit ? exit : exitLater);
-
+  mocha.files = files;
+  runner = mocha.run(program.exit ? exit : exitLater);
+  
+}
```
Comment (jbnicolai) [inline bin/_mocha]: I'd add a blank line after the `}`.

**Comment by jbnicolai** [comment]
LGTM

**Comment by aryeguy** [comment]
Added newline.

**Comment by jbnicolai** [comment]
:+1: thanks, @gaye. I didn't merge but used `git am` to squash both commits together.

**Comment by leebyron** [comment]
Thanks so much for fixing this @gaye and @jbnicolai! Any chance of deploying a point release on npm which includes this change?

**Comment by shannonmoeller** [comment]
Will this be released soon?

**Comment by mysterycommand** [comment]
:+1:

**Comment by sapegin** [comment]
:+1:

**Comment by abelnation** [comment]
:+1:

**Comment by danielstjules** [comment]
As a note, this went out with 2.3.0 :)

**Pull request closed without merging**